features/support/paths.rb

```diff
@@ -269,6 +274,10 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
       plan = Plan.find_by_name!($1)
       edit_polymorphic_path([:admin, plan])
 
+    when /^the edit page for admin service plan "([^"]*)"$/
```

Comment:
The previous step with the expression `/^the edit page for plan "([^"]*)"$/` can cover this scenario, so a new step is not necessary. **BUT!** the trick is that it will find the **first** plan with the defined name - the reason why we should avoid creating different plans (even of the same type) with the same names.